ci/parse-trivy-results.py

```diff
@@ -0,0 +1,121 @@
+"""Script to parse results from Trivy scan.
```

Comment:
You seem to have a whole bunch of trailing spaces in this file, please fix:
```
ci/parse-trivy-results.py:7:70: C0303: Trailing whitespace (trailing-whitespace)
ci/parse-trivy-results.py:9:68: C0303: Trailing whitespace (trailing-whitespace)
ci/parse-trivy-results.py:103:64: C0303: Trailing whitespace (trailing-whitespace)
ci/parse-trivy-results.py:105:0: C0303: Trailing whitespace (trailing-whitespace)
ci/parse-trivy-results.py:108:74: C0303: Trailing whitespace (trailing-whitespace)
```

I suggest setting your editor to automatically trim whitespace.